### PR TITLE
Time dependent BCs + iterative solver for penalty method

### PR DESCRIPTION
**Pull request by ksstar**
Wrong request

### CODE REVIEW

**Comment by giovastabile** [comment]
@ksstar  could you please do a rebase? There was an open PR from @sokratiag that I closed now.

Giovanni

File: src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.C [hunk 1]
```diff
@@ -419,41 +419,28 @@ void steadyNS::projectPPE(fileName folder, label NU, label NP, label NSUP)
             D_matrix = laplacian_pressure(NPmodes);
         }
 
-        word bc1_str = "BC1_" + name(liftfield.size()) + "_" + name(
```
Comment (giovastabile) [inline src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.C:422]: In this file there was a number of changes and I think you overrided the last changes

File: src/ITHACA_FOMPROBLEMS/unsteadyNS/unsteadyNS.C [hunk 2]
```diff
@@ -61,17 +61,34 @@ unsteadyNS::unsteadyNS(int argc, char* argv[])
...
+    M_Assert(timedepbcMethod == "yes" || timedepbcMethod == "no",
+    	     "The BC method can be set to yes or no");
     offline = ITHACAutilities::check_off();
     podex = ITHACAutilities::check_pod();
     supex = ITHACAutilities::check_sup();
 }
 
 // * * * * * * * * * * * * * * * Member Functions  * * * * * * * * * * * * * //
 
-void unsteadyNS::truthSolve(List<scalar> mu_now)
```
Comment (giovastabile) [inline src/ITHACA_FOMPROBLEMS/unsteadyNS/unsteadyNS.C:74]: also here I think there were problems with previous versions.

File: src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.C (hunk 1 above)
Comment (ksstar) [reply src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.C:422]: BC1_matrix and bc2tensor are not of need for the PPE method, so those don't have to be calculated in the offline phase

**Review by giovastabile — COMMENTED** [review]
I think we should review everything with a Skype Call because I think you override a number of previous naming conventions.

Giovanni

**Comment by giovastabile** [comment]
I closed it because from my understanding the new one includes also these modifications

**Pull request closed without merging**